docs/content/en/docs/implementing/tasks-non-k8s-apps.md

```diff
@@ -65,21 +105,42 @@ spec:
     workloadVersion: "1.0.0"
 ```
 
-Applying this file will cause Keptn to create a Job and a Pod and run the `KeptnTaskDefinition`.
+TODO: This file does not match what I see in the API Reference.
+See specific comments in the `KeptnTask` reference page.
+When we resolve those issues, I will modify this file appropriately.
 
-`kubectl get keptntasks` and `kubectl get pods` will show the current status of the jobs.
+You can then apply this YAML file with the following command:
 
-## Running More KeptnTasks
+```yaml
+kubectl --apply test-task.yaml
+```
 
-For subsequent KeptnTask runs, the `KeptnTask` name needs to be unique, update the follow fields:
+Applying this file causes Keptn to create a Job and a Pod
+and run the executables defined
+in the associated `KeptnTaskDefinition` resource.
+
+Use the following commands to show the current status of the jobs:
+
+```shell
+kubectl get keptntasks 
+kubectl get pods
+```
+
+## Re-run the KeptnTask
+
+For subsequent KeptnTask runs,
+the `KeptnTask` name and version fields must be unique,
+so copy the `KeptnTask` file you have and update the following fields:
 
 - `name`
 - `spec.appVersion`
 - `spec.workloadVersion`
```

Comment:
There was a mismatch with the v1alpha3 spec which has now been resolved (see https://github.com/keptn/lifecycle-toolkit/issues/2120). The `spec.*` fields are removed now, defining the app & workload details only inside `spec.context` will work for v1alpha3

Reply:
Okay, this is new information.  I was basically just moving the existing page from the "Tutorials" section to the "User Guides" section so carrying through the information that was there.  I can fix this.

Reply:
This content has now been reconciled with the revised v1alpha3 spec